kvdb/bolt_test.go

```diff
@@ -0,0 +1,93 @@
+package kvdb
```

Comment:
Scope creep but we could also add a dump and check into these tests.

Reply:
Yes considered that, but a dump of bbolt is not that easy, because it needs to go through all buckets recursively. As this test is extra already, left the final assertions out for now.